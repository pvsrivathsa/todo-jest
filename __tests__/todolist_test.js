import React from 'react';
import TodoList from '../src/components/TodoList';
import renderer from 'react-test-renderer';

/* Snapshot */

test('should render the todo list component', () => {
	const todoList = [{
		id: 1,
		text: "Attend Chennai JS Meetup",
		completed: false
	}]
	const component = renderer.create(
		<TodoList todos={todoList} onTodoClick={jest.fn()}/>
	);

	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
})