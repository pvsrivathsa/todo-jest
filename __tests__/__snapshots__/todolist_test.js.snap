exports[`test should render the todo list component 1`] = `
<ul>
  <li
    onClick={[Function]}
    style={
      Object {
        "textDecoration": "none",
      }
    }>
    Attend Chennai JS Meetup
  </li>
</ul>
`;
